doc/api/module.md

```diff
@@ -1556,6 +1556,56 @@ Running `node --import 'data:text/javascript,import { register } from "node:modu
 or `node --import ./import-map-sync-hooks.js main.js`
 should print `some module!`.
 
+## Module Hooks Reflection
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+### `module.loadModule(specifier, parentURL[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `specifier` {string} The URL of the module to load.
+* `parentURL` {string} The module importing this one.
+* `options` {Object} Optional
+  * `importAttributes` {Object} An object whose key-value pairs represent the
+    attributes for the module to import.
+* Returns: {Object}
```

Comment:
This returns a Promise, but I didn't find a good example on how to document the object properties.

Eg: https://github.com/nodejs/node/blob/bd3f27166b37cfde9349d31501cf0aa3401c7b61/doc/api/readline.md?plain=1#L536

Reply:
And maybe you could add more information what could make this promise to rejects, like specifier not found, etc...